main/jlp/net/http_api.cpp

```
@@ -684,6 +687,71 @@ esp_err_t hello_get(httpd_req_t* req) {
+  bool ok = true;
+  const char* err = nullptr;
+
+  char q[160];
+  if (httpd_req_get_url_query_str(req, q, sizeof(q)) == ESP_OK) {
+    char val[80];
+    if (httpd_query_key_value(q, "stop", val, sizeof(val)) == ESP_OK) {
+      stream_client_stop();
+    } else if (httpd_query_key_value(q, "start", val, sizeof(val)) == ESP_OK) {
+      std::string host = sk_server().host;
+      uint16_t soak_port = 5004;
+      if (httpd_query_key_value(q, "host", val, sizeof(val)) == ESP_OK) host = val;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)